libusb/hid.c

```diff
@@ -565,13 +567,18 @@ int HID_API_EXPORT hid_exit(void)
 /**
  * Requires an opened device with *claimed interface*.
  */
-static void fill_device_info_usage(struct hid_device_info *cur_dev, libusb_device_handle *handle, int interface_num)
+static void fill_device_info_usage(struct hid_device_info *cur_dev, libusb_device_handle *handle, int interface_num, uint16_t report_descriptor_size)
 {
 	unsigned char data[4096];
```

Comment:
I recommend to use https://github.com/libusb/hidapi/blob/65b9e712b1006366f646fad3b9d5d8e9de2dcfae/hidapi/hidapi.h#L97 from teh Get_Descriptor branch instead of repeating this value multiple times

Reply:
> from the Get_Descriptor branch

My current intention is to update this code for `get_descriptor` branch, once this PR is in master.
This branch is going to be merged to master sooner than `get_descriptor`.